{"name": "test_get_nonexistent_user", "status": "failed", "statusDetails": {"message": "AssertionError: {\"code\":1,\"type\":\"error\",\"message\":\"User not found\"}\nassert 'User not found' == 'Пользователь не найден'\n  \n  - Пользователь не найден\n  + User not found", "trace": "self = <tests.test_user_operations.TestUserOperations object at 0x000001D9161C1CA0>\nuser_page = <pages.user_page.UserPage object at 0x000001D9161E41A0>\n\n    def test_get_nonexistent_user(self, user_page):\n        with allure.step(\"Получение несуществующего пользователя\"):\n            response = user_page.get_user(\"Ryan Gosling\")\n            assert response.status_code == 404, response.text\n    \n            with allure.step(\"Прикрепление данных после теста\"):\n                allure.attach(response.text, name=\"Текст овтета\", attachment_type=allure.attachment_type.JSON)\n                allure.attach(str(response.headers), name=\"Заголовки\", attachment_type=allure.attachment_type.JSON)\n    \n            if response.headers.get('Content-Type') == 'application/json':\n>               assert response.json()['message'] == \"Пользователь не найден\", response.text\nE               AssertionError: {\"code\":1,\"type\":\"error\",\"message\":\"User not found\"}\nE               assert 'User not found' == 'Пользователь не найден'\nE                 \nE                 - Пользователь не найден\nE                 + User not found\n\ntest_user_operations.py:83: AssertionError"}, "steps": [{"name": "Получение несуществующего пользователя", "status": "failed", "statusDetails": {"message": "AssertionError: {\"code\":1,\"type\":\"error\",\"message\":\"User not found\"}\nassert 'User not found' == 'Пользователь не найден'\n  \n  - Пользователь не найден\n  + User not found\n", "trace": "  File \"C:\\Users\\bbuld\\PycharmProjects\\PetShop\\tests\\test_user_operations.py\", line 83, in test_get_nonexistent_user\n    assert response.json()['message'] == \"Пользователь не найден\", response.text\n"}, "steps": [{"name": "Прикрепление данных после теста", "status": "passed", "attachments": [{"name": "Текст овтета", "source": "c8c928f0-6fe5-4066-a57a-e9c9705d8e7a-attachment.json", "type": "application/json"}, {"name": "Заголовки", "source": "16bf009f-8989-44cf-b5fb-535255da20ef-attachment.json", "type": "application/json"}], "start": 1719729975744, "stop": 1719729975745}], "start": 1719729974659, "stop": 1719729975746}], "start": 1719729974659, "stop": 1719729975747, "uuid": "cb76d389-e324-4728-8df8-b6a1fcae7536", "historyId": "920462bc36800d833457c42b1bd38699", "testCaseId": "920462bc36800d833457c42b1bd38699", "fullName": "tests.test_user_operations.TestUserOperations#test_get_nonexistent_user", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('user_page', 'user_data', 'created_user')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_operations"}, {"name": "subSuite", "value": "TestUserOperations"}, {"name": "host", "value": "sanya"}, {"name": "thread", "value": "16872-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_operations"}]}